src/wp-includes/theme.php

```diff
@@ -2079,24 +2079,41 @@ function wp_update_custom_css_post( $css, $args = array() ) {
  * Since version 3.4 the TinyMCE body has .rtl CSS class.
  * It is a better option to use that class and add any RTL styles to the main stylesheet.
  *
+ * @param array|string $stylesheet Optional. Stylesheet name or array thereof, relative to theme root.
+ *                                 Defaults to 'editor-style.css'
+ *
+ * @param string $rtl_mode Optional. Whether if the site in RTL mode. Should the RTL style replace the default sheet or be added as well'.
```

Comment:
DocBlock issue: This docblock line should be consistently indented with the other param.